src/script/ui/viewportObserver.ts

```diff
@@ -113,23 +111,3 @@ export const viewportObserver = {
   removeElement,
   trackElement,
 };
-
-export const useViewPortObserver = (elementRef?: HTMLElement, defaultIsVisible: boolean = false): boolean => {
```

Comment:
This hook didn't really make sense. It only manipulate DOM elements. React component are tailored to manipulate DOM elements.

Instead the `InViewport` Component has been used